### DISCUSSION TITLE
If the player runs MinecraftServer.stopCleanly and does not successfully shut down the server, is there any solution to fix it?

### DISCUSSION THREAD

**Discussion by baka4n** [Minestom/Minestom#2245] [Help]
Just like this title

**Comment by mudkipdev** [comment]
It does successfully shut down, but it doesn't stop the process. You can use [`System#exit`](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/System.html#exit(int)) for that.

**Reply by baka4n** [reply]
If you need to close it and then start it, such as a restart command, then how to start it again

**Reply by baka4n** [reply]
Repetition
Minecraft.init
An error will be reported
